src/bin/juliainstaller.rs

```diff
@@ -234,17 +256,23 @@ pub fn main() -> Result<()> {
     }
 
     let mut install_choices = InstallChoices {
-        backgroundselfupdate: 0,
-        startupselfupdate: 1440,
+        backgroundselfupdate: args.background_selfupdate_interval.parse().unwrap(),
+        startupselfupdate: args.startup_selfupdate_interval.parse().unwrap(),
         symlinks: false,
-        modifypath: true,
+        modifypath: args.add_to_path.eq("yes"),
```

Comment:
I wonder whether clap has some way to make the Rust type here a bool, and then in clap specify which string value is mapped to `true` and `false`?

Reply:
I had to mess about with `BoolishValueParser`s -- and the whole thing had to be wrapped in `Option<bool>` to avoid it from becoming a flag. I think the solution now is a good one.